src/parser/parser.ee.ts

```diff
@@ -80,6 +86,15 @@ export async function parseStream({
       return result.value;
     }
 
+    case 'text/csv': {
+      const hexString = binaryStream.toString('hex');
+      const linesHex = hexString.split('0a');
```

Comment:
can you explain this? this is not straightforward, and I did not have this in the ansearch source code

Reply:
aka make a comment in your code

Reply:
I initially used 'utf8'. However I checked the review before and you mentioned "It's as simple as doing buffer.toString('hex') and then parsing out the \n into an array ". So I used 'hex' encoding. And in hexadecimal, the newline character \n is represented as 0a.

I was wondering if there's a specific requirement for using 'hex' over 'utf8'

Reply:
there is not, this is a brainfart from me sorry

use utf8